apps/andi/assets/webpack.config.js

```diff
@@ -55,5 +55,6 @@ module.exports = (env, options) => ({
   plugins: [
     new MiniCssExtractPlugin({ filename: '../css/app.css' }),
     new CopyWebpackPlugin({patterns: [{ from: 'static/', to: '../' }]})
-  ]
+  ],
+  stats: 'errors-only',
```

Comment:
does this shorten the startup message to only print if there's an error?

Reply:
Yes, it specifically gets rid of all the webpack output which can flood the logs during tests. I don't think this was always printing out before, but with some of the dependency upgrades it became more problematic.